src/javaRosa.js

```diff
@@ -826,6 +826,7 @@ define([
         var widget = widgets.base(mug, options);
         var $input = $("<textarea></textarea>")
             .attr("name", widget.id)
+            .attr("id", widget.id)
```

Comment:
These id additions are not necessarily required, but it helps with debugging: https://github.com/ichord/At.js/blob/ee088630480ebd0681b02296d0be30ae0c27ff9f/dist/js/jquery.atwho.js#L261

It'll either be the id or some unreadable id that you can't associate with a widget easily.